AlpsSDK/RemoteNotificationManager.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  RemoteNotificationManager.swift
+//  AlpsSDK
+//
+//  Created by Wen on 07.11.17.
+//  Copyright © 2017 Alps. All rights reserved.
+//
+
+import Foundation
+import UIKit
```

Comment:
is `UIKit` required?

Reply:
Yes, you are right it was linked to previous iOS version. 
I will remove UIKIt